Vizzuality/trase
# Create exception for indonesia country layer

## Asana

https://app.asana.com/0/inbox/1196306894124277/1200905903763449/1200905678407901

## Description

We had different layers for indonesia country of production in different contexts. Wood pulp is an exception and has a different layer

Extra:

Update contact link

## Testing instructions

Go to the tool and select indonesia-wood pulp and then country of production. You should see the different zones
Go to the tool and select indonesia-palm oil and then country of production. You should see the whole country
